x-pack/plugins/uptime/server/graphql/monitor_states/resolvers.ts

```diff
@@ -6,13 +6,13 @@
 
 import { CreateUMGraphQLResolvers, UMContext } from '../types';
 import { UMServerLibs } from '../../lib/lib';
-import { UMResolver } from '../../../common/graphql/resolver_types';
+import { UMResolver } from '../../../../../legacy/plugins/uptime/common/graphql/resolver_types';
 import {
   GetMonitorStatesQueryArgs,
   MonitorSummaryResult,
   StatesIndexStatus,
-} from '../../../common/graphql/types';
-import { CONTEXT_DEFAULTS } from '../../../common/constants/context_defaults';
+} from '../../../../../legacy/plugins/uptime/common/graphql/types';
+import { CONTEXT_DEFAULTS } from '../../../../../legacy/plugins/uptime/common/constants/context_defaults';
```

Comment:
(Optional) Not sure about the legacy codebase also still relying on these common types, but maybe they can be moved to NP now?

Reply:
We'll move `common` over with `public` - I think we'd like to get server merged sooner rather than later because we will have some deps holding us back on the client side.